tests/test_resolvers.py

```diff
@@ -20,4 +20,4 @@ def test_sanitize(self):
         self.assertEqual(c.c, "___fooo___")
         self.assertEqual(c.d, c.b)
         self.assertEqual(c.e, "_f")
-        self.assertEqual(c.f, " fooo")
+        self.assertEqual(c.f, "fooo")
```

Comment:
This change is unrelated to dataclass issues. It looks like the `sanatize_filename` function behavior is different from expected.